src/aat/java/uk/gov/hmcts/ccd/fta/exception/FunctionalTestException.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.ccd.fta.exception;
+
+public class FunctionalTestException extends RuntimeException {
+
+    private final String validationMessage;
```

Comment:
field redundant - super.getMessage() gives the same

Reply:
👍